config/openshift/rbac.yaml

```diff
@@ -9,4 +9,4 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: ingress-node-firewall-daemon
-  namespace: ingress-node-firewall-system
+  namespace: openshift-ingress-node-firewall
```

Comment:
was that intentional ?

Reply:
yes, within OCP, we want our product to start with `openshift-`.

Reply:
ok the ns is hardcoded in many place including e2e we will need to clean that up based on the env we are running on "runtime" I will look at next